### DISCUSSION TITLE
How to properly pass Automounted USB Drive to docker images

### DISCUSSION THREAD

**Discussion by koalacurioso** [IceWhaleTech/CasaOS#660] [General]
Hi, I'm trying CasaOS: I would like to use it with Syncthing sharing files from an USB drive. 

I see that my KINGSTON drive is auto mounted in /media/devmon/KINGSTONE
If I pass to docker this path Syncthing can read but not write on it. 
Can someone tell me if I need to change something in the automount script?

**Comment by ahnkle** [comment]
I also want syncthing to use an external drive but when it is connected it is not visible to syncthing.
